src/components/StakingVaults/DeFiEarn.tsx

```diff
@@ -19,7 +19,6 @@ type DefiEarnProps = {
   positionTableProps?: Omit<PositionTableProps, 'searchQuery'>
   includeEarnBalances?: boolean
   includeRewardsBalances?: boolean
-  header?: JSX.Element
```

Comment:
blocking: we still need this 
<img width="1657" alt="Screenshot 2025-03-24 at 20 19 06" src="https://github.com/user-attachments/assets/96ff7480-f5f8-44fe-aefd-b0725dd6129f" />

Reply:
Done in [8db94c1](https://github.com/shapeshift/web/pull/9086/commits/8db94c1ebd64971e4a6f3dd422478112132f863f)